project-backend/Controllers/BidController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using project_backend.Models.BidController.AddBid;
+using project_backend.Models.JobController.AddJob;
+using project_backend.Providers.BidProvider;
+using System.Linq;
+using project_backend.Models.Utils;
+using Microsoft.AspNetCore.Http;
+using project_backend.Models.BidController.EditBid;
+using project_backend.Models.Exceptions;
+using project_backend.Models.Bid;
+using project_backend.Models;
+
+namespace project_backend.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
```

Comment:
Since you're not using endpoint specific routes, you can say here [controller]/[action]

Reply:
That wouldn't enable me to make a post and a put on the root(bid/) though. Or would it?

Reply:
That's right, it wouldn't allow you to do that. If that's the behavior you want then that's fine :)